.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
Shouldn't we just update the rest of this to be the content from https://www.toptal.com/developers/gitignore/api/vim,emacs,vs,python,node,macos ?

Note that removes docker because `#!! ERROR: docker is undefined. Use list command to see defined gitignore types !!#`

Reply:
Still a thing